catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/transformations/EmbeddedMetacardsHandler.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.metacard.workspace.transformations;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.MetacardImpl;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.codice.ddf.catalog.ui.metacard.workspace.transformer.WorkspaceTransformation;
+import org.codice.ddf.catalog.ui.metacard.workspace.transformer.WorkspaceTransformer;
+
+/**
+ * This partial implementation of {@link WorkspaceTransformation} is intended to transform embedded
```

Comment:
It's a partial implementation because it implements some methods, but not others. An abstract base class would also work for this purpose, but multiple abstract classes cannot be extended by one class. These default methods require no state, so an abstract class is not needed.